### DISCUSSION TITLE
[Local Message] Model gpt-4o does not exist. 模型不存在, 或者您没有获得体验资格.

### DISCUSSION THREAD

**Discussion by xiaoqi798** [binary-husky/gpt_academic#2101] [Q&A]
小白求问，配置了网络 输入了api 但是还是不能用，秘钥是4.0gpt 购买了GPT 对应的api 还要付费吗？
![界面](https://github.com/user-attachments/assets/cc4c1730-e719-4967-af94-1117d38b60c2)
![体验资格](https://github.com/user-attachments/assets/f4307b3c-7033-464f-b984-b83c986982ac)
![未知错误](https://github.com/user-attachments/assets/ee3ccbc1-65b8-48fb-9bc6-dba249da90b3)

**Comment by binaryYuki** [comment]
如果是测试 apikey 建议使用`gpt-4o` 从当前返回来看是 api 返回MODEL NOT FOUND的错误 也就是这个 apikey 无法使用`gpt-4`模型
